kdl_parser_py/package.xml

```diff
@@ -19,16 +20,16 @@
   <url type="repository">https://github.com/ros2/kdl_parser</url>
   <url type="bugtracker">https://github.com/ros2/kdl_parser/issues</url>
 
-  <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
-  <buildtool_depend>python-catkin-pkg</buildtool_depend>
-
   <build_depend version_gte="1.3.0">orocos_kdl</build_depend>
   <build_depend>urdf</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>urdf</run_depend>
-  <run_depend>urdfdom_py</run_depend>
-  <run_depend>python_orocos_kdl</run_depend>
+  <exec_depend version_gte="1.3.0">orocos_kdl</exec_depend>
+  <exec_depend>urdf</exec_depend>
+  <exec_depend>urdfdom_py</exec_depend>
+  <exec_depend>python_orocos_kdl</exec_depend>
```

Comment:
Unfortunately, this is going to be a problem.  We don't have `python_orocos_kdl` ported to ROS 2 at this time.  See https://github.com/ros2/geometry2/pull/360#issuecomment-754651422 for some of my thoughts on how we can get this ported and enabled for ROS 2.

Reply:
Sounds like you have these two options already mapped out well. Is there already a preference regarding which one to go for? Has someone already picked this up or is there a timeline? Can I help something with this? Although I probably couldn't help with the second option since I have no Windows machine available and no experience with ros on Windows...

Reply:
Nobody has picked it up as far as I know.  So help there would be welcome (it would unblock both this and that geometry2 PR).

As far as which way to go, I would probably choose option 1.  It's the most straightforward to do.  Keep in mind that we will have to get it working for Windows (and macOS) there as well, but that should be easier than making a choco package.

Reply:
Okay, I think I can find some time to look into option 1.